### PR TITLE
Update Version Information in README

### PR DESCRIPTION
**Pull request by stsantilena**
Removed the `rustc` version dependency information. Also, made some formatting changes and added some new badges.
Screen Shot 2022-09-23 at 1 58 07 PM

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # LogDNA Agent
 
-[![Rustc Version 1.46+]][rustc]
-[Join us at the LogDNA community forum]: https://community.logdna.com
+![agent version](https://img.shields.io/badge/Version-3.6.0-blue.svg)
+[![made-with-rust](https://img.shields.io/badge/Made%20with-Rust-1f425f.svg)](https://www.rust-lang.org/)
+
+Join us at the LogDNA community [forum](https://community.logdna.com).
```
Comment (stsantilena) [inline docs/README.md:6]: @alfeng6 not sure if we want to leave this or change given the branding?

File: docs/README.md [hunk 2]
```diff
@@ -141,7 +140,7 @@ You can also obtain the image and review our tagging scheme on [DockerHub](https
 
 ### Building Agent Binary on Linux
 
-The agent requires `v1.42+` of rustc [cargo-make](https://github.com/sagiegurari/cargo-make) to build. If the proper versions of rustc and cargo are installed; then simply run the following command to build the agent:
+The agent requires rustc [cargo-make](https://github.com/sagiegurari/cargo-make) to build. If rustc and cargo are installed; then simply run the following command to build the agent:
```
Comment (dkhokhlov) [inline docs/README.md:143]: re: installed
i would link this to rustup https://www.rust-lang.org/tools/install.

Comment (stsantilena) [reply docs/README.md]: Updated.

File: docs/README.md [hunk 3]
```diff
@@ -141,7 +140,7 @@ You can also obtain the image and review our tagging scheme on [DockerHub](https
 
 ### Building Agent Binary on Linux
 
-The agent requires `v1.42+` of rustc [cargo-make](https://github.com/sagiegurari/cargo-make) to build. If the proper versions of rustc and cargo are installed; then simply run the following command to build the agent:
+The agent requires rustc [cargo-make](https://github.com/sagiegurari/cargo-make) to build. If rustc and cargo are [installed](https://www.rust-lang.org/tools/install); then simply run the following command to build the agent:
```
Comment (alfeng6) [inline docs/README.md:143]:
```suggestion
The agent requires rustc [cargo-make](https://github.com/sagiegurari/cargo-make) to build. If rustc and cargo are [installed](https://www.rust-lang.org/tools/install), simply run the following command to build the agent:
```

**Review by alfeng6 — CHANGES_REQUESTED** [review]
just a small grammar change. Other than that, looks great!

File: docs/README.md (hunk 3 above)
Comment (stsantilena) [reply docs/README.md]: Updated.

**Review by alfeng6 — APPROVED** [review]
looks good

**Pull request merged**